index.js

```diff
@@ -52,33 +51,19 @@ export async function upload (firstPath, opts) {
 /**
  * Print out all the uploads in the current space.
  * @param {object} opts
+ * @param {boolean} [opts.raw]
  * @param {boolean} [opts.json]
  * @param {boolean} [opts.shards]
  */
-export async function list (opts) {
+export async function list (opts = {}) {
   const client = await getClient()
   let count = 0
   let res
   do {
     res = await client.capability.upload.list()
```

Comment:
Need to pass pagination info as options

Reply:
oh - we actually don't support pagination options here because it automatically fetches all the results. maybe worth revisiting, but I think out of scope for this PR?

Reply:
Yes this needs a fix because now it'll output the same results over and over forever as soon as you upload more than the default page size items.

Would you mind opening an issue?

Reply:
ohhhh got it, totally right, should be an easy fix honestly, lemme see if I can get it into this real quick...

Reply:
ok opened #48 for this, but it's a pretty bad bug and an easy fix (I think) so gonna take a sec while my head is here to try to fix